styles.css

```diff
@@ -0,0 +1,542 @@
+
+* {
+  scrollbar-width: none;
+  list-style: none;
+  box-sizing: border-box;
+}
+
+*::-webkit-scrollbar {
+  display: none;
+}
+
+.check,
+.check * {
+  border: solid 1px rgb(0, 0, 0);
+}
+
+#header {
+  /* position: absolute; */
+  width: 100vw;
+  left: 0;
+  z-index: 10;
+  border-bottom: 2px solid gray;
+  background-color: white;
+}
+
+#header a {
+  color: black;
+  text-decoration: none;
+}
+
+#header a:hover {
+  color: black;
+  text-decoration: none;
+}
+
+#header-nav-bar {
+  display: flex;
+  flex-direction: row;
+  column-gap: 37%;
+}
+
+.session-container {
+  display: none;
+}
+
+.secondary-nav {
+  display: none;
+}
+
+#menu {
+  position: absolute;
+  z-index: 1;
+}
+
+.close-menu-icon {
+  position: absolute;
+  display: none;
+  cursor: pointer;
+
+  /* opacity: 0; */
```

Comment:
Kindly don't leave any uncommented code in your codebase, in this way, you will keep your codebase clean, neat and organized.

Reply:
Thank you for the suggestion, it's done!